conda/core/index.py

```diff
@@ -58,6 +60,8 @@ def get_index(channel_urls=(), prepend=True, platform=None,
         unknown = True
 
     channel_urls = calculate_channel_urls(channel_urls, prepend, platform, use_local)
+    del LAST_CHANNEL_URLS[:]
+    LAST_CHANNEL_URLS.extend(channel_urls)
```

Comment:
This whole `LAST_CHANNEL_URLS` thing is pretty ugly.  The `get_index()` and `install_actions()` functions really aren't used by conda itself anymore though.  Once conda-build is able to adapt to new conda APIs, these can be gotten rid of.  So this is a bridge measure until that time.